explainers/hidden-content-explainer.md

```diff
@@ -349,7 +365,7 @@ for the beforematch event, and the details element already has a toggle event
 which would act like the beforematch event. This could be implemented either by
 adding an attribute to the details element saying that the collapsed content
 should be searchable, or just by making it searchable as-is because find-in-page
-is not specced.
+is not specced rigitly enough to forbid this.
```

Comment:
... because find-in-page spec does not specify this behavior

Reply:
done